x-pack/plugins/canvas/public/components/workpad_page/workpad_static_page/workpad_static_page.scss

```diff
@@ -0,0 +1,13 @@
+.canvasStaticPage {
+  z-index: initial;
+  box-shadow: 0 0 7px 0 rgba(0, 0, 0, 0.2);
```

Comment:
@include euiBottomShadowFlat(opacity: .4)

Same thing here.

Reply:
You could always make this a small mixin or separate selector so you're not repeating yourself.

Reply:
Done, it got unified